applications/GeoMechanicsApplication/tests/test_partially_saturated/test_saturated_below_phreatic_level_pw_triangle3N/ProjectParameters_stage1.json

```diff
@@ -0,0 +1,191 @@
+      },
+      {
+        "python_module"                : "apply_scalar_constraint_table_process",
+        "kratos_module"                : "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name"                 : "ApplyScalarConstraintTableProcess",
+        "Parameters": {
+          "model_part_name"            : "PorousDomain.WATER_PRESSURE_Bottom",
+          "variable_name"              : "WATER_PRESSURE",
+          "is_fixed"                   : true,
+          "table"                      : 0,
+          "fluid_pressure_type"        : "Phreatic_Multi_Line",
+          "gravity_direction"          : 1,
+          "out_of_plane_direction"     : 2,
+          "x_coordinates"              : [ 0.0,  1.0],
+          "y_coordinates"              : [-2.0, -2.0],
+          "z_coordinates"              : [ 0.0,  0.0],
+          "specific_weight"            : 10000.0
+        }
+      },
+      {
+        "python_module"                : "apply_scalar_constraint_table_process",
+        "kratos_module"                : "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name"                 : "ApplyScalarConstraintTableProcess",
+        "Parameters": {
+          "model_part_name"            : "PorousDomain.WATER_PRESSURE_Top",
+          "variable_name"              : "WATER_PRESSURE",
+          "is_fixed"                   : true,
+          "table"                      : 0,
+          "fluid_pressure_type"        : "Phreatic_Multi_Line",
+          "gravity_direction"          : 1,
+          "out_of_plane_direction"     : 2,
+          "x_coordinates"              : [ 0.0,  1.0],
+          "y_coordinates"              : [-2.0, -2.0],
+          "z_coordinates"              : [ 0.0,  0.0],
+          "specific_weight"            : 10000.0
+        }
+      }
+    ],
+    "loads_process_list": [
+      {
+        "python_module"                : "apply_vector_constraint_table_process",
+        "kratos_module"                : "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name"                 : "ApplyVectorConstraintTableProcess",
+        "Parameters": {
+          "model_part_name"            : "PorousDomain.gravity",
+          "variable_name"              : "VOLUME_ACCELERATION",
+          "active"                     : [false, true, false],
+          "value"                      : [0.0, -9.81, 0.0],
+          "table"                      : [0, 0, 0]
+        }
+      }
+    ],
+    "auxiliar_process_list": [
+     {
+        "python_module"                : "geo_extrapolate_integration_point_values_to_nodes_process",
+        "kratos_module"                : "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name"                 : "GeoExtrapolateIntegrationPointValuesToNodesProcess",
+        "Parameters": {
+          "model_part_name"            : "PorousDomain",
+          "list_of_variables"          : ["TOTAL_STRESS_TENSOR", "CAUCHY_STRESS_TENSOR"]
+        }
```

Comment:
We aren't checking the values produced by this process, are we? If that is correct, I would suggest to remove this process from the test, to keep it as simple as possible.